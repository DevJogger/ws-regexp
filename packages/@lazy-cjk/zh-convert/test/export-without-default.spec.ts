import _m0 = require('../');
import _m1 from '../';

test(`import check without default`, () =>
{
	let actual = Object.keys(_m0)
		.filter(n => n !== 'default')
	;
	let expected = Object.keys(_m1);

	expect(actual).toStrictEqual(expected);
	//expect(actual).toBeInstanceOf(Date);
	expect(actual).toMatchSnapshot();

});
