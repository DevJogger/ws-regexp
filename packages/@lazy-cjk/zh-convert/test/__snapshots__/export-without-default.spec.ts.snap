// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import check without default 1`] = `
Array [
  "cn2tw",
  "tw2cn",
]
`;
